zerver/lib/bugdown/__init__.py

```diff
@@ -560,12 +560,19 @@ def handleMatch(self, match):
 if settings.VOYAGER:
     path_to_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
                                  'prod-static', 'serve', 'third', 'gemoji', 'images', 'emoji', '*.png')
+    path_to_unicode_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
+                                         'prod-static', 'serve', 'third', 'gemoji', 'images',
+                                         'emoji', 'unicode', '*.png')
 else:
     path_to_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
                                  # This should be the root
                                  'static', 'third', 'gemoji', 'images', 'emoji', '*.png')
+    path_to_unicode_emoji = os.path.join(os.path.dirname(__file__), '..', '..', '..',
+                                         # This should be the root
+                                         'static', 'third', 'gemoji', 'images', 'emoji', 'unicode', '*.png')
```

Comment:
Oops, I didn't make clear the fact that `STATIC_ROOT` doesn't actually work, at least in that form, because it's sometimes an absolute and sometimes a relative path:
```
(zulip-venv) tabbott@zaset:~/zulip$ git grep STATIC_ROOT
tools/update-prod-static:subprocess.check_call(['mv', os.path.join(settings.STATIC_ROOT, 'source-map'),
zerver/lib/bugdown/__init__.py:emoji_tree = os.path.join(settings.DEPLOY_ROOT, settings.STATIC_ROOT,
zproject/settings.py:# STATIC_ROOT even for dev servers.  So we only use
zproject/settings.py:        STATIC_ROOT = 'prod-static/serve'
zproject/settings.py:        STATIC_ROOT = 'static/'
zproject/settings.py:        STATIC_ROOT = '/home/zulip/prod-static'
zproject/settings.py:        STATIC_ROOT = 'prod-static/serve'
```
which is why I suggested the slightly more messy code above.  An alternative approach would be to fix `STATIC_ROOT` to be more consistent (e.g. by calling `os.path.abspath()` on it), which I think would be better since that would solve this whole category of issue.